modules/sharedaddy/sharing-service.php

```diff
@@ -867,6 +867,13 @@ function sharing_display( $text = '', $echo = false ) {
 			$sharing_content .= implode( '', $parts );
 			$sharing_content .= '<li class="share-end"></li></ul>';
 
+			// Link to customization options if user can manage them.
+			if ( current_user_can( 'manage_options' ) ) {
+				$link_text        = esc_html__( 'Customize buttons', 'jetpack' );
```

Comment:
With no dynamic elements to the string literal, we can just go with `__()`.

In WordPress, it's considered best practice to escape as late as possible, so in this case, we can leave `$link_text` and `$link_url` without any escaping for simplicity, and then escape when concatenating to `$sharing_content` (which also covers any later changes to those variables without worry).

VIP has some great docs around escaping: https://wpvip.com/documentation/vip-go/validating-sanitizing-and-escaping/#always-escape-late